pcap4j-core/src/main/java/org/pcap4j/core/Pcaps.java

```diff
@@ -218,6 +223,33 @@ public static PcapHandle openOffline(
 
   /**
    *
+   * @param filePath "-" means stdin
+   * @param precision PCAP_TSTAMP_PRECISION_NANO or PCAP_TSTAMP_PRECISION_MICRO
+   * @return a new PcapHandle object.
+   * @throws PcapNativeException
+   */
+  public static PcapHandle openOfflineWithTstampPrecision(
```

Comment:
This kind of method name looks like a traditional C method. Overloading openOffline is more like Java way and refined.